lib/SPIRV/SPIRVBuiltinHelper.cpp

```diff
@@ -103,7 +103,10 @@ Value *BuiltinCallMutator::doConversion() {
       Builder.Insert(addCallInst(CI->getModule(), FuncName, ReturnTy, Args,
                                  &Attrs, nullptr, Mangler.get()));
   NewCall->copyMetadata(*CI);
-  NewCall->setAttributes(CI->getAttributes());
+  if (CI->hasFnAttr("fpbuiltin-max-error")) {
+    auto Attr = CI->getFnAttr("fpbuiltin-max-error");
+    NewCall->addFnAttr(Attr);
```

Comment:
Should we also call `setAttributes()` to copy all the other attributes, not just `fpbuiltin-max-error`?

Reply:
In addition to Viktoria's comment, like this we also discard calling convention if it was not a default one.

Reply:
I am trying to NOT introduce any new logic here. This PR aims to only correct the bug introduced by PR #2056

Reply:
I do not think copyMetadata overwrites existing metadata. So we should be fine here.

Thanks